--- gimp-print-5.0.0-alpha1/configure.ac.orig	2004-01-17 16:08:53.000000000 +0100
+++ gimp-print-5.0.0-alpha1/configure.ac	2004-01-17 16:11:06.027654776 +0100
@@ -740,7 +740,7 @@
 fi
 AM_CONDITIONAL(BUILD_MODULES, test x$MODULE = xtrue)
 
-AM_CONDITIONAL(BUILD_GENPPD_STATIC, test x"$enable_static" = xyes)
+AM_CONDITIONAL(BUILD_GENPPD_STATIC, false)
 
 if test "$enable_static" = no -a "$BUILD_CUPS" = yes ; then
   AC_MSG_WARN([--enable-static is disabled, but CUPS is being built.])
