--- gimp-print-4.2.7/configure.in.orig	2004-07-19 22:39:48.390628128 +0200
+++ gimp-print-4.2.7/configure.in	2004-07-19 22:41:50.073129576 +0200
@@ -95,7 +95,7 @@
 AM_CONFIG_HEADER(config.h)
 
 dnl Available languages (translations)
-ALL_LINGUAS="en_GB sv no fr da pl sk el nl es pt de ja cs nb uk hu"
+ALL_LINGUAS="en_GB sv fr da pl sk el nl es pt de ja cs nb uk hu"
 AC_SUBST(ALL_LINGUAS)
 
 dnl Compiler options and definitions
