--- gimp-print-4.2.2/src/escputil/escputil.c.orig	Fri Aug 23 01:48:52 2002
+++ gimp-print-4.2.2/src/escputil/escputil.c	Mon Feb 10 15:34:36 2003
@@ -473,12 +473,12 @@
         if (printer == NULL)
           strcpy(command, "lpr -l");
 	else
-          sprintf(command, "lpr -P%s -l", printer);
+          snprintf(command, sizeof(command), "lpr -P%s -l", printer);
         }
       else if (printer == NULL)
 	strcpy(command, "lp -s -oraw");
       else
-	sprintf(command, "lp -s -oraw -d%s", printer);
+	snprintf(command, sizeof(command), "lp -s -oraw -d%s", printer);
 
       if ((pfile = popen(command, "w")) == NULL)
 	{
